GameOfThrones/Models/Character.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace GameOfThrones.Models
+{
+    public class Character
+    {
+        public int CharacterId { get; set; }
```

Comment:
Why `CharacterId` and not just `Id`? Sometimes there is a need to be explicit however, I'm not sure this is one of those occassions